include/video.h

```diff
@@ -34,25 +34,24 @@ typedef enum {
 
 typedef void (*GFX_CallBack_t)( GFX_CallBackFunctions_t function );
 
-#define GFX_CAN_8   0x0001
-#define GFX_CAN_15  0x0002
-#define GFX_CAN_16  0x0004
-#define GFX_CAN_32  0x0008
-
-#define GFX_LOVE_8  0x0010
-#define GFX_LOVE_15 0x0020
-#define GFX_LOVE_16 0x0040
-#define GFX_LOVE_32 0x0080
-
-#define GFX_RGBONLY 0x0100
-#define GFX_DBL_H   0x0200 /* double-width  flag */
-#define GFX_DBL_W   0x0400 /* double-height flag */
-
-#define GFX_SCALING		0x1000
-#define GFX_HARDWARE	0x2000
-
-#define GFX_CAN_RANDOM  0x4000 //If the interface can also do random access surface
-#define GFX_UNITY_SCALE 0x8000 /* turn of all scaling in render.cpp */
+constexpr uint16_t GFX_CAN_8  = 1 << 0;
+constexpr uint16_t GFX_CAN_15 = 1 << 1;
+constexpr uint16_t GFX_CAN_16 = 1 << 2;
+constexpr uint16_t GFX_CAN_32 = 1 << 3;
+
+constexpr uint16_t GFX_LOVE_8  = 1 << 4;
+constexpr uint16_t GFX_LOVE_15 = 1 << 5;
+constexpr uint16_t GFX_LOVE_16 = 1 << 6;
+constexpr uint16_t GFX_LOVE_32 = 1 << 7;
```

Comment:
Next thing: get rid of `CAN` and `LOVE` 🤦🏻 What is `LOVE`? Something enabled? Some optional preference? I have no idea by looking at these names...

Reply:
Gone; turns out nothing was using these!